docs/user-guide.md

```diff
@@ -114,8 +115,9 @@ In this example, both methods are in the scope of `@NullMarked`, so plain `Strin
 means "a reference to a `String` that can't be null". `@Nullable String` continues
 to mean "a reference to a `String` that can be null". Tools should warn you if you
 try to pass a "reference to a `String` that can be null" to `spaceIndex`, since its
-argument can't be null, and indeed it will throw NullPointerException if given a
+argument can't be null, and indeed it will throw a NullPointerException if given a
 null argument.
+TODO: warning only expected if call is also in `@NullMarked` context, per earlier comment?
```

Comment:
An earlier comment (at the end of `Types and nullness`) called out calls from non-NullMarked code. So for the hypothetical call of `spaceIndex`, we could make the two cases explicit here.